### DISCUSSION TITLE
SFR1M44-FU

### DISCUSSION THREAD

**Discussion by thecientificdr** [keirf/flashfloppy#476] [General]
I can flash SFR1M44-FU image for Embroidery machine if somebody have to SFR1M44-U100?

**Comment by keirf** [comment]
I don't understand the question but I think those two models are the same hardware.

**Reply by thecientificdr** [reply]
I want somebody extract the Firmware from FU and pass to me to Flash in U100. 

Thanks!

**Reply by keirf** [reply]
This isn't possible as the factory firmware is read protected.
